### PR TITLE
Clarify recommended customization of build scripts

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@markwilkie @chcosta PTAL

**Comment by markwilkie** [comment]
Thanks @tmat for doing this.  LGTM

File: Documentation/ArcadeSdk.md [hunk 1]
```diff
@@ -93,15 +93,23 @@ Having a common output directory structure makes it possible to unify MicroBuild
...
 
-Most repos additionally create repo specific build scripts which reference the `eng/common/build.ps1` or `eng/common/build.sh` files.  They provide default arguments to those scripts but pass along any additional arguments you specify.
+- eng/common/[tools.ps1](https://github.com/dotnet/arcade/tree/master/eng/common/tools.ps1)|[tools.sh](https://github.com/dotnet/arcade/tree/master/eng/common/tools.sh)
 
-Repo script examples include:
+  Defined global variables and functions used in all builds scripts. This include helpers that install .NET SDK, invoke MSBuild, locate Visual Studio, report build telemetry, etc.
```
Comment (chcosta) [inline Documentation/ArcadeSdk.md:104]:
```suggestion
  Defines global variables and functions used in all builds scripts. This includes helpers that install .NET SDK, invoke MSBuild, locate Visual Studio, report build telemetry, etc.
```

File: Documentation/ArcadeSdk.md [hunk 2]
```diff
@@ -93,15 +93,23 @@ Having a common output directory structure makes it possible to unify MicroBuild
...
+
+- eng/common/[CIBuild.cmd](https://github.com/dotnet/arcade/tree/master/eng/common/CIBuild.cmd)|[cibuild.sh](https://github.com/dotnet/arcade/tree/master/eng/common/cibuild.sh)
```
Comment (chcosta) [inline Documentation/ArcadeSdk.md:106]: May help to additionally clarify what the "-ci" option is  and its purpose.  ie, we want local builds and ci builds to behave identically, but there are some behaviors which are required by ci and we ensure those are specified when "ci" is defined.

Comment (tmat) [reply Documentation/ArcadeSdk.md:106]: I'll leave that for another time :)

Perhaps the best place to add more info would be here:
https://github.com/dotnet/arcade/blob/master/eng/common/tools.ps1#L5

**Pull request merged**